vendor/github.com/gregjones/httpcache/httpcache.go

```diff
@@ -471,7 +471,7 @@ func cloneRequest(r *http.Request) *http.Request {
 	r2 := new(http.Request)
 	*r2 = *r
 	// deep copy of the Header
-	r2.Header = make(http.Header)
+	r2.Header = make(http.Header, len(r.Header))
```

Comment:
things under vendor come from other repos and should not be changed directly (and are verified to match the original source in CI)